mfkdf2/src/derive/factors/password.rs

```diff
@@ -23,15 +23,10 @@ pub fn password(password: impl Into<String>) -> MFKDF2Result<MFKDF2Factor> {
     return Err(MFKDF2Error::PasswordEmpty);
   }
   let strength = zxcvbn(&password, &[]);
-  let strength = strength.guesses().ilog2() as f64;
 
   Ok(MFKDF2Factor {
     factor_type: FactorType::Password(Password { password }),
-    // TODO (autoparallel): This is confusing, should probably put an Option here. This pattern
-    // appears in other factors and it's because of the refactoring done. The factors have a
-    // "state" assiociated to them basically (in that they are "setup" or not).
```

Comment:
This comment still applies though I think you know this.

The `MFKDF2Factor` can be in a sort of `Uninit` state or `Init` state. Certain items should likely be excluded from the `Uninit` and `Init`

This is a future problem and just for type safety though.